types/src/transaction/mod.rs

```diff
@@ -91,7 +91,8 @@ impl Transaction {
     }
 
     pub fn get_hash(&self) -> CryptoHash {
-        let bytes = borsh::to_vec(&self).expect("Failed to deserialize");
+        #[allow(clippy::expect_used)]
+        let bytes = borsh::to_vec(&self).expect("Failed to serialize");
```

Comment:
All that Borsh::to_vec is inherited from other code bases, where it was expected not to fail. 

Why? @frol